package.json

```diff
@@ -7,7 +7,9 @@
   "scripts": {
     "dev": "vite --host",
     "build": "tsc && vite build",
-    "check": "biome check .",
+    "test": "npm run test:static && npm run test:browsers",
+    "test:static": "biome check .",
+    "test:browsers": "playwright test",
```

Comment:
Nitpick: The original `check` script is a format/lint checker, and not really a kind of test. Tests would commonly be something like `test:e2e` (playwright) and `test:unit` (vitest+testing-library).

Reply:
@PixnBits Let me know if this sounds like a reasonable change. I'd love to see this PR get merged, and happy to assist if needed.

Reply:
sorry for the delay
I'd argue that static code analysis is testing too ;)
but I'll update this to separate out tests from other checks :+1:

Reply:
updated in 3c9088b